target/linux/ath79/dts/qca9531_comfast_cf-wr752ac.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-wr752ac", "qca,qca9531";
+	model = "COMFAST CF-WR752AC";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &lan;
+		led-failsafe = &lan;
+		led-upgrade = &lan;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins &led_lan &led_wlan2g>;
+
+		lan: lan {
+			label = "cf-wr752ac:green:lan";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "netdev";
+			trigger-sources = <&eth0>;
+			default-state = "on";
+		};
+
+		wlan2g {
+			label = "cf-wr752ac:red:wlan2g";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+			linux,default-trigger = "phy1tpt";
+			panic-indicator;
+		};
+
+		wlan5g {
+			label = "cf-wr752ac:blue:wlan5g";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+			linux,default-trigger = "phy0tpt";
+		};
+
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		button {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	watchdog {
+		compatible = "linux,wdt-gpio";
+
+		gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		hw_algo = "toggle";
+		hw_margin_ms = <1200>;
+		always-running;
+	};
+};
+
+
+&pinmux {
+	led_lan: pinmux_lan {
+		pinctrl-single,bits = <0x0c 0x0 0xff000000>;
+	};
+
+	led_wlan2g: pinmux_wlan2g {
+		pinctrl-single,bits = <0x08 0x0 0xff000000>;
+	};
+};
+
+&pcie0 {
+	status = "okay";
+};
+
+&spi {
+	status = "okay";
+	num-cs = <1>;
+
+	flash@0 {
+		compatible = "winbond,w25q128", "jedec,spi-nor" ;
+		reg = <0>;
+		spi-max-frequency = <25000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			uboot: partition@0 {
+				label = "u-boot";
+				reg = <0x000000 0x010000>;
+				read-only;
+			};
+
+			art: partition@10000 {
+				label = "art";
+				reg = <0x010000 0x010000>;
+				read-only;
+			};
+
+			firmware: partition@20000 {
+				compatible = "denx,uimage";
+				label = "firmware";
+				reg = <0x020000 0xfd0000>;
+			};
+
+			nvram: partition@ff0000 {
+				label = "nvram";
+				reg = <0xff0000 0x010000>;
+				read-only;
+			};
+		};
+	};
+};
+
+&uart {
+	status = "okay";
+};
+
+&eth0 {
+	status = "okay";
+	phy-handle = <&swphy4>;
+	mtd-mac-address = <&art 0x0>;
+};
+
+&eth1 {
+	// "Disable" unpopulated GMAC1
```

Comment:
I do not think this comment is needed.

Reply:
Are you sure that everyone knows that eth1 has to be disabled this unorthodox way, since there is no physical eth1 present, and the whole mess with eth0 <-> eth1 name switching ...
Precedent case is in file "qca9531_glinet_gl-ar300m-lite.dts".